website/docs/reference/source-configs.md

```diff
@@ -172,6 +251,53 @@ sources:
 </File>
 
 
+#### Configure a source with an `event_time`
+
+<VersionBlock lastVersion="1.8">
+
+Configuring an [`event_time`](/reference/resource-configs/event-time) for a source is only available in dbt Cloud Versionless or dbt Core versions 1.9 and later.
+
+</VersionBlock>
+
+<VersionBlock firstVersion="1.9">
+
+To configure a source with an `event_time`, specify the `event_time` field in the source configuration. This field is used to represent the actual timestamp of the event, rather than something like a loading date.
+
+For example, if you had a source table called `clickstream` in the `events` source, you can use the timestamp for each event in the `event_timestamp` column as follows:
+
+<File name='dbt_project.yml'>
+
+```yaml
+sources:
+  events:
+    clickstream:
+      +event_time: event_timestamp
+```
+</File>
+
+In this example, the `event_time` is set to `event_timestamp`, which has the exact time each clickstream event happened.
+Not only is this required for the [incremental microbatching strategy]((/docs/build/incremental-microbatch)), but when you compare data across [CI and production](/docs/deploy/advanced-ci#considerations) environments, dbt will use `event_timestamp` to filter and match data by this event-based timeframe, ensuring that only overlapping timeframes are compared.
```

Comment:
Hiya @mirnawong1 

This link takes me to a page not found.
I think this is the correct link: /docs/build/incremental-microbatch (I think it might be the double brackets causing the issue)

Kind Regards
Natalie